Server/server/models/index.js

```diff
@@ -0,0 +1,38 @@
+
+
```

Comment:
try to see how you could make changes to this file so eslint does not flag it.

Reply:
Done, no more flagging